### ISSUE TITLE
Prepare links for saved posts

### ISSUE THREAD

**Issue by dmtrjsg** [Joystream/pioneer#2117]
**AC**

- [ ] Component prep work for tooltips - every resource has a separaTe component:
[a member ✅](https://www.figma.com/file/GlgN8uBRtvtMJtiOsdtDF7/Pioneer-Design?node-id=5674%3A156591)
[a Forum thread ✅](https://www.figma.com/file/GlgN8uBRtvtMJtiOsdtDF7/Pioneer-Design?node-id=5674%3A156591)
[a Forum post ✅](https://www.figma.com/file/GlgN8uBRtvtMJtiOsdtDF7/Pioneer-Design?node-id=5674%3A156792)
[a proposal ✅](https://www.figma.com/file/GlgN8uBRtvtMJtiOsdtDF7/Pioneer-Design?node-id=5674%3A156263)
[a proposal discussion entry](https://www.figma.com/file/GlgN8uBRtvtMJtiOsdtDF7/Pioneer-Design?node-id=5674%3A156792) ✅ ⚠️ Use the same design with threads
[an opening ✅](https://www.figma.com/file/GlgN8uBRtvtMJtiOsdtDF7/Pioneer-Design?node-id=5674%3A157177)
[an application ✅](https://www.figma.com/file/GlgN8uBRtvtMJtiOsdtDF7/Pioneer-Design?node-id=5674%3A157350)

- [ ] Links are prepared

- [ ] For each of the tooltips we need to query query node 

** ⛔  Out of scope**

Blocked by #1880 
- [ ] blog posts
- [ ] blog threads
